editor/project_manager.cpp

```diff
@@ -109,21 +109,25 @@ void ProjectDialog::_set_message(const String &p_msg, MessageType p_type, InputT
 	}
 }
 
+static bool is_zip_file(Ref<DirAccess> p_d, const String &p_path) {
+	return p_path.ends_with(".zip") && p_d->file_exists(p_path);
+}
+
 String ProjectDialog::_test_path() {
 	Ref<DirAccess> d = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
+	const String base_path = project_path->get_text();
```

Comment:
Any reason we shouldn't `strip_edges()` here and not have to deal with doing the logic twice with and without stripping below?

Reply:
That's a good point, I think I just didn't think of it, will take a look

Reply:
Right, on Linux I can have trailing and leading space:
```
$ file "  LICENSE.txt  " 
  LICENSE.txt  : ASCII text
```